deeppavlov/core/common/params.py

```diff
@@ -55,9 +55,8 @@ def from_params(params: Dict, **kwargs) -> Component:
             log.exception(e)
             raise e
 
-    try:
-        cls_name = config_params.pop('name')
-    except KeyError:
+    cls_name = config_params.pop('name', None)
+    if not cls_name:
```

Comment:
How is this better?

Reply:
Shouldn't throw an exception, if there is a valid checking key method like "pop".